src/scripts/main.js

```diff
@@ -1,5 +1,29 @@
 'use strict';
 
 document.addEventListener('click', (e) => {
-  // write code here
+  if (e.target.classList.contains('wall')) {
+    const x = e.clientX;
+    const y = e.clientY;
```

Comment:
The names of the variables 'x' and 'y' are not descriptive enough. It's better to give them more meaningful names, like 'clickX' and 'clickY' to make it clear that they represent the coordinates of the click event.